rts/lra/service-base/src/main/java/io/narayana/lra/LRAData.java

```diff
@@ -41,11 +41,12 @@
     private long startTime;
     private long finishTime;
     private int httpStatus;
+    private String lraType;
 
     public LRAData() {}
 
     public LRAData(URI lraId, String clientId, LRAStatus status, boolean isTopLevel, boolean isRecovering,
-                    long startTime, long finishTime, int httpStatus) {
+                    long startTime, long finishTime, int httpStatus, String lraType) {
```

Comment:
the status indicates the terminal state so the lraType data is not required.

Reply:
Status would not verify that the failed records are moved to another location. Is there any other way to do this?

Reply:
I addressed this in one of my earlier comments.

Reply:
okay, thanks